api/app/Models/PoolCandidate.php

```diff
@@ -270,6 +270,20 @@ public function filterByEquity(Builder $query, array $equity): Builder
         return $query;
     }
 
+    public function scopePoolCandidateStatus(Builder $query, ?string $poolCandidateStatus): Builder
+    {
+        if(empty($poolCandidateStatus)){
+            return $query;
+        }
+
+        if($poolCandidateStatus == 'AVAILABLE'){
+            $query->where('pool_candidate_status', 'AVAILABLE');
+            return $query;
+        }
+
+        return $query;
+    }
+
```

Comment:
In addition to a flexible dynamic scope, it would be handy to have a scope that [doesn't include any args at all](https://laravel.com/docs/9.x/eloquent#local-scopes).

```
    public function scopeIsAvailable(Builder $query): Builder
    {
            return $query->where('pool_candidate_status', 'AVAILABLE');
    }

```